### PR TITLE
[docs] [ISSUE 11380] Update code example and description

### LINKED ISSUE #11380: CLI in TTL config is not clear
**Is your enhancement request related to a problem? Please describe.**
image

Not quite understand what does "N/A", "100" mean in a separate CLI in https://pulsar.apache.org/docs/en/next/admin-api-namespaces/#configure-message-ttl

Is it the returned value?

**Describe the solution you'd like**
Give clear and concise description for the above values.

### PR DESCRIPTION
**Pull request by sijia-w**
Fixes #11380 

It seems that the changed two file share common content on the usage of message retention policies. Do we need to keep the examples of the commands in both files?

### CODE REVIEW

**Comment by sijia-w** [comment]
@Jennifer88huang Please help review the changes. Thank you.

**Review by Jennifer88huang — APPROVED** [review]

File: site2/docs/admin-api-namespaces.md [hunk 1]
```diff
@@ -400,8 +392,7 @@ admin.namespaces().unloadNamespaceBundle(namespace, bundle)
 
 #### Split namespace bundles
 
-Each namespace bundle can contain multiple topics and each bundle can be served by only one broker. 
-If a single bundle is creating an excessive load on a broker, an admin splits the bundle using this command permitting one or more of the new bundles to be unloaded thus spreading the load across the brokers.
+One namespace bundle can contain multiple topics but can be served by only one broker. If a single bundle is creating an excessive load on a broker, an admin can split the bundle using the command below, permitting one or more of the new bundles to be unloaded, thus spreading the load across the brokers.
```
Comment (Huanli-Meng) [inline site2/docs/admin-api-namespaces.md:395]: I think "balancing the load across brokers" is more appropriate.

**Comment by sijia-w** [comment]
@Huanli-Meng Thanks for your comment. Updated. PTAL.

**Review by Huanli-Meng — APPROVED** [review]
LGTM

**Review by sijie — APPROVED** [review]

**Pull request merged**